### ISSUE TITLE
How can i change colors in term?

### ISSUE THREAD

**Issue by kden416** [ioquatix/script-runner#93]
package ignore theme dark or light option

**Comment by kden416** [comment]
changing .atom/packages/script-runner/styles/script-runner.less has no effect
same .atom/packages/script-runner/node_modules/xterm/dist/xterm.css

**Comment by ioquatix** [comment]
Yeah, xterm changed completely. I don't know how you style it now. I will need to investigate, my apologies.

**Comment by kden416** [comment]
Strange, but on first system i have nodeVersion": "8.9.4", xterm 3.3.0 and other system same, but on first sytem terminal in atom looks fine and on second system looks very ugly.

**Comment by kden416** [comment]
i think addon xterm-webfont can help us.

**Comment by ioquatix** [comment]
Do you mind submitting a PR?

**Comment by ioquatix** [comment]
Also can you share some pictures, because I don't know what it looks like on your system, what is good and what is ugly. It may help other users too, if they have the same problem.

**Comment by kden416** [comment]
if i fix it, i submit PR, but i'm not JS programmer and it's difficult for me.


this is first system screenshot:
![screenshot_20180508_142400](https://user-images.githubusercontent.com/1554073/39754626-76722eea-52cb-11e8-8670-95499d9dd3ef.png)

and second:
![screenshot_20180508_142226](https://user-images.githubusercontent.com/1554073/39754606-66f9ef20-52cb-11e8-90d2-658a5068fc0c.png)
how can you see fonts...

**Comment by ioquatix** [comment]
I see, yeah, it looks pretty crappy.
